samples/excel/89-preview-apis/range-hyperlink.yaml

```diff
@@ -0,0 +1,180 @@
+id: excel-89-preview-apis-range-hyperlink-
+name: Range hyperlink
+description: 'Create, update, and remove a hyperlink for a range.'
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.7
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+
+        $("#create-url-hyperlink").click(() => tryCatch(createUrlHyperlink));
+        $("#create-document-hyperlink").click(() => tryCatch(createDocumentHyperlink));
+        $("#update-hyperlink").click(() => tryCatch(updateHyperlink));
+        $("#remove-hyperlink").click(() => tryCatch(removeHyperlink));
+
+        async function createUrlHyperlink() {
+            await Excel.run(async (context) => {
+                const activeSheet = context.workbook.worksheets.getActiveWorksheet();
```

Comment:
Here and elsewhere: should you be using the active sheet or getting the sheet by name?